lib/reig/context.cpp

```diff
@@ -181,7 +232,15 @@ namespace reig {
             if (mConfig.mWindowsTextured) {
                 render_rectangle(currentWindow.drawData, bodyBox, mConfig.mWindowBackgroundTexture);
             } else {
-                render_rectangle(currentWindow.drawData, bodyBox, mConfig.mWindowBackgroundColor);
+                int thickness = 1;
+                render_rectangle(currentWindow.drawData, internal::decrease_rect(bodyBox, thickness),
+                                 mConfig.mWindowBackgroundColor);
+
+                auto frameColor = mConfig.mTitleBackgroundColor;
+                auto frameRectangles = internal::get_rect_frame(bodyBox, thickness);
+                for (const auto& frameRect : frameRectangles) {
+                    render_rectangle(currentWindow.drawData, frameRect, frameColor);
```

Comment:
A desaturated frame color can be used for inactive windows

Reply:
![image](https://user-images.githubusercontent.com/4677693/39095527-b5f0e846-464a-11e8-9172-d371304a664b.png)
![image](https://user-images.githubusercontent.com/4677693/39095530-c21e933e-464a-11e8-9982-e3a158739d95.png)